sections/semantics-textlevel.include

```diff
@@ -2550,25 +2492,23 @@ Linux demo 2.6.10-grsec+gg3+e+fhs6b+nfs+gr0501+++p3+c4a+gr2b-reslog-v6.189 #1 SM
     Here, the <{sub}> element is used to represent the subscript that identifies the
     variable in a family of variables:
 
-    <pre highlight="html">
-&lt;p&gt;The coordinate of the &lt;var&gt;i&lt;/var&gt;th point is
-(&lt;var&gt;x&lt;sub&gt;&lt;var&gt;i&lt;/var&gt;&lt;/sub&gt;&lt;/var&gt;, &lt;var&gt;y&lt;sub&gt;&lt;var&gt;i&lt;/var&gt;&lt;/sub&gt;&lt;/var&gt;).
-For example, the 10th point has coordinate
-(&lt;var&gt;x&lt;sub&gt;10&lt;/sub&gt;&lt;/var&gt;, &lt;var&gt;y&lt;sub&gt;10&lt;/sub&gt;&lt;/var&gt;).&lt;/p&gt;
-    </pre>
+    <xmp highlight="html">
+      <p>The coordinate of the <var>i</var>th point is (<var>x<sub><var>i</var></sub></var>, <var>y<sub><var>i</var></sub></var>).
+      For example, the 10th point has coordinate (<var>x<sub>10</sub></var>, <var>y<sub>10</sub></var>).</p>    
+    </xmp>
   </div>
 
   Mathematical expressions often use subscripts and superscripts. Authors are encouraged to use
   MathML for marking up mathematics, but authors may opt to use <{sub}> and
   <{sup}> if detailed mathematical markup is not desired. [[!MATHML]]
 
   <div class="example">
-    <pre highlight="html">
-&lt;var&gt;E&lt;/var&gt;=&lt;var&gt;m&lt;/var&gt;&lt;var&gt;c&lt;/var&gt;&lt;sup&gt;2&lt;/sup&gt;
-    </pre>
-    <pre highlight="html">
-f(&lt;var&gt;x&lt;/var&gt;, &lt;var&gt;n&lt;/var&gt;) = log&lt;sub&gt;4&lt;/sub&gt;&lt;var&gt;x&lt;/var&gt;&lt;sup&gt;&lt;var&gt;n&lt;/var&gt;&lt;/sup&gt;
-    </pre>
+    <xmp highlight="html">
```

Comment:
do these need to be two distinct blocks of code?

Reply:
Fixed.